sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentAnalysisClientOptions.cs

```diff
@@ -57,5 +58,21 @@ internal static string GetVersionString(ServiceVersion version)
                 _ => throw new NotSupportedException($"The service version {version} is not supported."),
             };
         }
+
+        /// <summary>
+        /// Add headers and query parameters that are considered safe for logging or including in
+        /// error messages by default.
+        /// </summary>
+        private void AddLoggedHeadersAndQueryParameters()
+        {
+            Diagnostics.LoggedHeaderNames.Add(Constants.OperationLocationHeader);
+            Diagnostics.LoggedHeaderNames.Add("apim-request-id");
+            Diagnostics.LoggedHeaderNames.Add("Strict-Transport-Security");
+            Diagnostics.LoggedHeaderNames.Add("x-content-type-options");
+            Diagnostics.LoggedHeaderNames.Add("x-envoy-upstream-service-time");
+
+            Diagnostics.LoggedQueryParameters.Add("locale");
```

Comment:
should we add modelId, operationId, and resultId too?

Reply:
I could be wrong, but these IDs are not part of the query nor the headers, are they? I think they're contained in the endpoint.

Reply:
ohh you are right.
should we include the api version?

Reply:
Added the api version.